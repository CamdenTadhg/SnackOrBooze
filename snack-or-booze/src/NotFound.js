import React from 'react';
import {Card, CardBody, CardTitle} from 'reactstrap';
// import '.NotFound.css';

function NotFound() {
    return (
        <section>
            <Card>
                <CardBody>
                    We're sorry. That page doesn't exist.
                </CardBody>
            </Card>
        </section>
    )
}

export default NotFound;