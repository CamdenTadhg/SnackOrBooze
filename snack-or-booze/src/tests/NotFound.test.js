import React from 'react';
import {render} from '@testing-library/react';
import NotFound from "../NotFound";

it("renders without crashing", function() {
    render(<NotFound />);
});

it("matches snapshot", function() {
    const {asFragment} = render(<NotFound />);
    expect(asFragment()).toMatchSnapshot();
});

it('displays the correct content', function() {
    const {getByText} = render(<NotFound />);
    expect(getByText("We're sorry.", {exact: false})).toBeInTheDocument();
})