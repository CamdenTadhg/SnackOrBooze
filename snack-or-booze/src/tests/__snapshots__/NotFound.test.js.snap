// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <section>
    <div
      class="card"
    >
      <div
        class="card-body"
      >
        We're sorry. That page doesn't exist.
      </div>
    </div>
  </section>
</DocumentFragment>
`;
